examples/bridge-app/linux/main.cpp

```diff
@@ -604,6 +552,36 @@ EmberAfStatus HandleReadPowerSourceAttribute(DevicePowerSource * dev, chip::Attr
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+EmberAfStatus HandleReadTempMeasurementAttribute(DeviceTempSensor * dev, chip::AttributeId attributeId, uint8_t * buffer, uint16_t maxReadLength)
+{
+    if ((attributeId == ZCL_TEMP_MEASURED_VALUE_ATTRIBUTE_ID) && (maxReadLength == 2))
+    {
+        *(int16_t *) buffer = dev->GetMeasuredValue();
```

Comment:
Nothing guarantees the buffer is going to be 2-byte-aligned, as far as I know.  This should be using memcpy, I suspect.  Same for the bits below.

Reply:
Will there be any endian problems if I change this to a memcpy?

Also, is there a cleaner way that this should be using (probably in a future PR since it would apply to other existing code)?

Reply:
memcpy will just copy the bytes, which is what you want; the values are stored in native-endian.

And there is no better way right now....

Reply:
Got it, thanks.

Changed to memcpy.